src/server/getters/get-markets-awaiting-designated-reporting.ts

```diff
@@ -15,9 +14,10 @@ export function getMarketsAwaitingDesignatedReporting(db: Knex, designatedReport
     db.select().from("markets").where(queryData).whereNull("marketStateID").asCallback((err?: Error|null, marketsRows?: Array<MarketsRow>): void => {
         if (err) return callback(err);
         if (!marketsRows || !marketsRows.length) return callback(null);
-        each(marketsRows, (marketsRow: MarketsRow, nextMarketsRow: ErrorCallback): void => {
+        const marketsInfo: UIMarketsInfo = [];
+        marketsRows.map((marketsRow: MarketsRow): void => {
```

Comment:
`map` should return the value it implicitly creates an array.

```typescript
% [1, 2, 3].map((v) => v+10)
=> [11, 12, 13]
```

So in this case

```typescript
const marketsInfo: UIMarketsInfo = marketsRows.map(_.partialRight(reshapeMarketsRowToUIMarketInfo, []));
```
Will get you what you want, will also need to import lodash at the top for `_.partialRight`

```typescript
import * as _ from "lodash";
```